src/main/java/com/jcraft/jsch/Slf4jLogger.java

```diff
@@ -27,22 +41,30 @@ public boolean isEnabled(int level) {
 
   @Override
   public void log(int level, String message) {
+    log (level, message, null);
```

Comment:
Minor nit: there appears to be an extraneous space character between `log` and `(`.